src/core/CMakeLists.txt

```diff
@@ -14,6 +14,8 @@ add_library(
   halide2isl.cc
   halide_utils.cc
 
+  cpu/cpu_tc_executor.cc
```

Comment:
I'd make cpu executor an optional library, similar to CUDA.  The reason for that would be picking between "vanilla LLVM" and "Tapir" cpu executors and, eventually, between different CPU backends.

Reply:
yup, I can add here or in a followup PR depending on @salexspb need to have this in ```dev``` asap or not